app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  has_many :comments
+  has_many :likes
+
+  after_create :update_user_post_count
+  after_create :recent_comments
+
+  def update_user_post_count
+    author.increment!(:posts_counter) if author.present?
+  end
+
```

Comment:
- [ ] Please make sure to follow as I requested above `after_save.`

Reply:
> * [ ]  Please make sure to follow as I requested above `after_save.`

- [ ] Kindly implement this required changes from the previous reviewer. It is yet to be implemented. Kindly check the [differences between after_create and after_save](https://www.altcademy.com/questions/after-create-vs-after-save-in-rails#:~:text=after_create%20%2D%20works%20once%2C%20only%20after,create%20and%20ActiveRecord%23update%20method.) as explained to you by the previous reviewer.